public/app/features/migrate-to-cloud/onprem/ResourceDetailsModal.tsx

```diff
@@ -18,6 +18,16 @@ function getTMessage(errorCode: MigrateDataResponseItemDto['errorCode']): string
         'migrate-to-cloud.resource-details.error-messages.datasource-name-conflict',
         'There is a data source with the same name in the target instance. Rename one of them and try again.'
       );
+    case 'DATASOURCE_INVALID_URL':
+      return t(
+        'migrate-to-cloud.resource-details.error-messages.datasource-invalid-url',
+        'Data source must contain a valid URL.'
```

Comment:
nit: Maybe we can give this a "There is a data source whose URL..." like the other errors?

Reply:
@Proximyst updated, LMK what you think